### DISCUSSION TITLE
Changes to Autograder causing errors

### DISCUSSION THREAD

**Discussion by josh-test-student** [community/community#111482] [GitHub Education]
### Hi everyone,

In the last 3-4 days I've discovered that some of my students' repositories for an assignment are being set up differently than students that started the assignment a week ago. In particular, the way the autograder works seems to have changed quite significantly, causing some students to have failed tests now, when they would have been successful a few days ago.

For example, students who created their repositories 4 or 5 days ago have a folder called "classroom" with an `autograding.json` file, and the `classroom.yml` file was using `education/autograding@v1`, but now is using other actions like `education/autograding-io-grader@v1`. The `autograding-io-grader` fails to correctly grade input that was working fine with `autograding`. For example, my tests fail if my "Run command" includes `&&` to run multiple commands in sequence, where this was working fine before the change.

Was this an intended change, or did I do something in my GitHub Classroom assignment that caused this?

Thanks,
Josh

**Comment by danilocurvelo** [comment]
Same problem here. I'm actually including those files (`autograding.json` and `classroom.yml`) myself in the template repository.

**Comment by TUP-UTN-FRC-LCIII** [comment]
Same problem here!! Some assignments works and others no.
I see that exist code of test is 0 but action fail
